src/schemas/21-686C/schema.js

```diff
@@ -1,7 +1,25 @@
 import definitions from '../../common/definitions';
+import { countries, states50AndDC } from '../../common/constants';
 import schemaHelpers from '../../common/schema-helpers';
 import _ from 'lodash';
 
+const textRegex = '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*][\\w ]+$';
```

Comment:
why is this needed?

Reply:
Good question.  
From `EVSS DAM Form 686c_674 Updated version 17_14.xlsx`:
```
"For each character:
1. Any alphabetic lower or upper-case character
2.. No more than one consecutive space
3. The following Special Characters are not permitted (<, >, %, $, #, @, !, ^, &, *)
4. No numbers
5. Only valid values should be entered"
```

I have not tried submitting values that violate these rules to see if the service validates them.